CHANGELOG.md

```diff
@@ -126,6 +126,8 @@ These changes are available on the `master` branch, but have not yet been releas
 
 ### Fixed
 
+- Fixed `AttributeError` when user has unstable internet
+  ([#2301](https://github.com/Pycord-Development/pycord/pull/2301))
```

Comment:
Should go at the bottom of the caetegory

Reply:
Done